defaults/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
-
 # default user to apply the key to
-user: '{{ ansible_user_id }}'
+ssh_keys_ad_account: '{{ ansible_user_id }}'
+ssh_keys_github_accounts:
+  - '{{ ssh_keys_ad_account }}'
```

Comment:
My concern here is when the AD username differs from the GitHub username, the wrong Gihub account will get queried. At best a 404, at worst undesired ssh keys will get applied.

If the mapping ends up in vars/main.yml, then that can be referenced

Reply:
Yeah, sounds good. Thanks for catching that. I'll update it to use the hash instead.